plugin/trino-elasticsearch/src/test/java/io/trino/plugin/elasticsearch/BaseElasticsearchConnectorTest.java

```diff
@@ -499,6 +499,103 @@ public void testMixedArray()
                 "VALUES NULL, ARRAY['hello'], ARRAY['foo', 'bar']");
     }
 
```

Comment:
We should update existing tests that use `_meta.presto`. E.g.,  `testMixedArray`, `testArrayFields`.

Reply:
Done